jupyterhub/schema.yaml

```diff
@@ -2220,6 +2216,37 @@ properties:
         type: [integer, "null"]
         description: See the `--max-age` flag.
 
+  defaultTolerations:
+    type: object
+    required: [enabled]
+    description: |
+      Default [Toleration objects](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#toleration-v1-core)
+      applied to all Pods.  If your cluster has a [PodTolerationRestriction 
+      admission-controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction)
+      then you may need to configure it to accept these tolerations, 
+      or disable the defaultTolerations.
```

Comment:
There are two sets of tolerations, one with `value: core` and one with `value: user`.

User pods are the user servers and the image puller pods part of the image-puller daemonsets.

All other pods are considered core pods.